README.md

```diff
@@ -93,6 +95,18 @@ commands will create and activate the virtual environment:
  % pip install hypothesis
  % pip list
 ```
+
+> If you are using Python 3.12, please install `pip install -U git+https://github.com/pymtl/pymtl3@pymtl4.0-dev` and checkout py312-dev branch, difference is:
```

Comment:
Would this work for CGRA translation? The reason we use `git+https://github.com/tancheng/pymtl3.1@yo-struct-list-fix` is that it contains some specific fixes for us from @yo96 and @ptpan, I doubt whether the latest pymtl4.0 would work for us. Can you try CGRA (and tile) translation on python3.12 with pymtl4.0?

Reply:
Can it be proved by below 2 test cases pass?
1. CGRATemplateRTL_test.py in cgra/translate
2. TileVectorRTL_test.py in tile/translate

Failed for TileRTL_test.py in tile/translate by exception below, tried with pymtl3.1@yo-struct-list-fix and py37, same error.
```
FAILED         [100%]
tile/translate/TileRTL_test.py:69 (test_translate)
E   FileNotFoundError: [Errno 2] No such file or directory: 'pkg-config'
```